File: jscomp/test/inline_string_test.js
'use strict';


console.log("list");

console.log("list");

function f(param) do
  if (param ~= undefined) do
    return "Some";
  end else do
    return "None";
  end
end

console.log(--[ tuple ]--[
      f(3),
      "None",
      "Some"
    ]);

console.log(--[ tuple ]--[
      "A",
      "A"
    ]);

--[  Not a pure module ]--

File: jscomp/test/inline_string_test.lua
__console = {log = print};


__console.log("list");

__console.log("list");

function f(param) do
  if (param ~= nil) then do
    return "Some";
  end else do
    return "None";
  end end 
end end

__console.log(--[[ tuple ]]{
      f(3),
      "None",
      "Some"
    });

__console.log(--[[ tuple ]]{
      "A",
      "A"
    });

exports = {};
return exports;
--[[  Not a pure module ]]
